### DISCUSSION TITLE
添加资源报错

### DISCUSSION THREAD

**Discussion by ohSteve** [ZuodaoTech/everyone-can-use-english#1139] [Q&A]
youtube视频可以打开但无法加载，
![IMG_20241020_224247_edit_293410452795155](https://github.com/user-attachments/assets/ccb2413d-2370-4ce8-99c5-04f66437550c)

**Comment by an-lee** [comment]
这个应该是因为你的网络没有全局代理，可以在 软件设置 / 高级设置 里把代理地址设置一下，让 Enjoy 也能走你的代理服务。

**Reply by ohSteve** [reply]
还是不太行，下面的网络状态也都正常的，用url有以上报错，主页推荐的视频下载没反应
